src/utils/time.ts

```diff
@@ -70,3 +71,25 @@ function quickFormat(timestamp: number, i18next: i18nType, formatRule: string) {
 function locale(i18next: i18nType) {
   return locales[i18next?.language]?.dateFns ?? locales['en-US']?.dateFns
 }
+
+export function formattedDuration(interval: Duration) {
+  const years = interval.years ?? 0
+  const months = interval.months ?? 0
+  const days = interval.days ?? 0
+
+  if (years === 0 && months === 0) {
+    if (days === 0) {
+      return i18n.t('duration', { context: 'lessThanADay' })
```

Comment:
optional nit, could save some characters if in each branch you do something like
```ts
tParams = { context: 'lessThanADay' }
```
and at the end do
```ts
return i18n.t('duration', tParams)
```

Reply:
Changed in 17d8e8dbd46d1f4288e43f2eb022cb4d0d7a410e!